### PR TITLE
fix: Markdownlint cleanup for ZH-CN

### PR DESCRIPTION
**Pull request by nschonni**
- MD014 dollar sign in code fences
- MD023 Headings at start of line
- Some MD034 bare links
- double backticks from conversion
- broken definition lists
- escaped OLs

### CODE REVIEW

File: files/zh-cn/games/anatomy/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ slug: Games/Anatomy
 
 具体情况取决于游戏本身。
 
-一些游戏通过**用户输入**来驱动这个循环。想象一下，你正在开发一种“大家来找茬”类型的游戏。这些游戏向用户**呈现**两张图片，游戏**接收**点击（或触摸）；将用户输入**解释**为成功，失败，暂停，菜单交互等。最后，游戏根据用户的输入**计算并更新**游戏场景。这种游戏是由用户的输入驱动，也就是说，它会在用户进行输入之后冻结画面，等待玩家进行新的输入。这是一种基于回合的游戏类型，它不需要每帧持续更新画面，只有当玩家作出反应时才会。
+一些游戏通过**用户输入**来驱动这个循环。想象一下，你正在开发一种"大家来找茬"类型的游戏。这些游戏向用户**呈现**两张图片，游戏**接收**点击（或触摸）；将用户输入**解释**为成功，失败，暂停，菜单交互等。最后，游戏根据用户的输入**计算并更新**游戏场景。这种游戏是由用户的输入驱动，也就是说，它会在用户进行输入之后冻结画面，等待玩家进行新的输入。这是一种基于回合的游戏类型，它不需要每帧持续更新画面，只有当玩家作出反应时才会。
```
Comment (yin1999) [inline files/zh-cn/games/anatomy/index.md:15]:
```suggestion
一些游戏通过**用户输入**来驱动这个循环。想象一下，你正在开发一种“大家来找茬”类型的游戏。这些游戏向用户**呈现**两张图片，游戏**接收**点击（或触摸）；将用户输入**解释**为成功，失败，暂停，菜单交互等。最后，游戏根据用户的输入**计算并更新**游戏场景。这种游戏是由用户的输入驱动，也就是说，它会在用户进行输入之后冻结画面，等待玩家进行新的输入。这是一种基于回合的游戏类型，它不需要每帧持续更新画面，只有当玩家作出反应时才会。
```

Hi @nschonni, we are used to use double quotes in Simplified Chinese documents (It's just right for us). So could we ignore this case in markdown lint?

Comment (nschonni) [reply files/zh-cn/games/anatomy/index.md:15]: Yeah, probably, this is part of a newer custom rule in the English repository. I didn't do a big replacement for those, just in the last few files

**Comment by nschonni** [comment]
@yin1999 i rebased out the smartquote changes, except for a few that just became code sections

File: files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md [hunk 2]
```diff
@@ -203,7 +203,7 @@ ls -l
 
 在您创建的某个地方的测试目录中使用它们，这样您就不会意外地删除任何重要的内容，使用下面的示例命令作为指导
 
-- `mkdir` —这将在您所在的当前目录中创建一个新目录，名称是您在命令名之后提供的。例如` mkdir my-awesome-website将创建一个新目录叫``my-awesome-website `.
+- `mkdir` —这将在您所在的当前目录中创建一个新目录，名称是您在命令名之后提供的。例如` mkdir my-awesome-website将创建一个新目录叫 `my-awesome-website`.
```
Comment (yin1999) [inline files/zh-cn/learn/tools_and_testing/understanding_client-side_tools/command_line/index.md:206]:
```suggestion
- `mkdir` —这将在您所在的当前目录中创建一个新目录，名称是您在命令名之后提供的。例如 `mkdir my-awesome-website` 将创建一个新目录叫 `my-awesome-website`。
```

File: files/zh-cn/web/api/canvas_api/tutorial/basic_animations/index.md [hunk 3]
```diff
@@ -681,6 +681,7 @@ function tmz() {
 
 ##### Output
 
+```html
```
Comment (yin1999) [inline files/zh-cn/web/api/canvas_api/tutorial/basic_animations/index.md:684]: This should be a rendered html table in markdown (not a code example).

And I've try to replace this with a `{{EmbedLiveSample}}` macro, but it doesn't work. Also, this `Snake Game` example seems to be removed in `en-US`. Maybe we'd better to remove this part (from line 453 to line 701)

File: files/zh-cn/web/api/ui_events/keyboard_event_key_values/index.md [hunk 4]
```diff
@@ -3074,7 +3074,7 @@ The values below are derived in part form a number of consumer electronics techn
 
 - [DTV Application Software Environment](http://atsc.org/standard/a100-dtv-application-software-environment-level-1-dase-1/) (part of the [ATSC](https://zh.wikipedia.org/wiki/ATSC) specification)
 - [Open Cable Application Platform 1.1.3](http://www.cablelabs.com/specifications/OC-SP-OCAP1.1.3-100603.pdf)
-- [ANSI/CEA-2014-B](http://www.ce.org/Standards/Standard-Listings/R7-Home-Network-Committee/CEA-2014-B-(ANSI%29.aspx): Web-based Protocol and Framework for Remote User Interface on UPnP™ Networks and the Internet
+- [ANSI/CEA-2014-B](<http://www.ce.org/Standards/Standard-Listings/R7-Home-Network-Committee/CEA-2014-B-(ANSI%29.aspx>): Web-based Protocol and Framework for Remote User Interface on UPnP™ Networks and the Internet
```
Comment (yin1999) [inline files/zh-cn/web/api/ui_events/keyboard_event_key_values/index.md:3077]: let's update this link (https://github.com/mdn/content/blob/db9797df4277b7f4c004f1a2b079b1e2f87bc8cb/files/en-us/web/api/ui_events/keyboard_event_key_values/index.md?plain=1#L3077):

```suggestion
- [ANSI/CEA-2014-B](https://shop.cta.tech/products/web-based-protocol-and-framework-for-remote-user-interface-on-upnp-networks-and-the-internet): Web-based Protocol and Framework for Remote User Interface on UPnP™ Networks and the Internet
```

File: files/zh-cn/web/api/webvtt_api/index.md [hunk 5]
```diff
@@ -99,62 +99,64 @@ In the case of each cue:
 
 我们还可以在.vtt 文件中放置注释，以帮助我们记住关于文件各部分的重要信息。这些应该在单独的行上，从字符串 NOTE 开始。您将在下一节中找到更多关于这些的信息。
 
-重要的是不要在提示中使用“额外的”空白行，例如在计时行和提示有效负载之间。WebVTT 是基于行;空白行将关闭提示。
+重要的是不要在提示中使用"额外的"空白行，例如在计时行和提示有效负载之间。WebVTT 是基于行;空白行将关闭提示。
```
Comment (yin1999) [inline files/zh-cn/web/api/webvtt_api/index.md:102]:
```suggestion
重要的是不要在提示中使用“额外的”空白行，例如在计时行和提示有效负载之间。WebVTT 是基于行;空白行将关闭提示。
```

**Review by yin1999 — COMMENTED** [review]
Thanks @nschonni, just some nits:

**Review by yin1999 — APPROVED** [review]
Thanks a lot @nschonni 👍

**Pull request merged**